src/olympia/amo/tests/test_storage_utils.py

```diff
@@ -33,21 +32,22 @@ def test_storage_walk():
         results = [(dir, set(subdirs), set(files))
                    for dir, subdirs, files in sorted(walk_storage(tmp))]
 
-        yield (eq_, results.pop(0),
```

Comment:
I couldn't see what difference having all these in a yield was doing (wouldn't pop always remove and return the first entry anyway?)